models/project-extension/src/category-recommendation-settings/generator.ts

```diff
@@ -0,0 +1,18 @@
+import { fake, Generator } from '@commercetools-test-data/core';
+import { createRelatedDates } from '@commercetools-test-data/utils';
+
+import type { TCategoryRecommendationSettings } from './types';
+
+const [getOlderDate, getNewerDate] = createRelatedDates();
+
+const generator = Generator<TCategoryRecommendationSettings>({
+  fields: {
+    createdAt: fake(getOlderDate),
+    id: fake((f) => f.string.uuid()),
+    updatedAt: fake(getNewerDate),
+    searchProperty: fake((f) => f.lorem.slug()),
```

Comment:
This is an enum of either `Attribute` or `ProductType`

Reply:
updated [97d95f9](https://github.com/commercetools/test-data/pull/562/commits/97d95f91fd46b7a4911c3a162c5a1bdbc56c649f)